app/konnect/gateway-manager/plugins/add-custom-plugin.md

```diff
@@ -0,0 +1,179 @@
+the {{site.konnect_short_name}} Custom Plugins API.
+
+If you need to update a schema for a plugin that has already been uploaded
+to {{site.konnect_short_name}}, there are a few considerations based on the type 
+of update. 
+See [Editing or deleting a custom plugin's schema](/konnect/gateway-manager/plugins/update-custom-plugin/) 
+for more information.
+
+{:.important}
+> **Caution**: Carefully test the operation of any custom plugins before deploying
+them to production. Kong is not responsible for the operation or support of any 
+custom plugins, including any performance impacts on your {{site.konnect_short_name}}
+or {{site.base_gateway}} deployments. 
+
+## Prerequisites
+
+* You have a custom {{site.base_gateway}} plugin with a `schema.lua` file, and no
+ `api.lua`, `daos.lua`, or `migration.lua` files.
+
+* Your custom plugin meets all other [requirements](/konnect/gateway-manager/plugins/#custom-plugins)
+for {{site.konnect_short_name}}.
+
+* The custom plugin can be written in any supported language, but the schema file must be in Lua.
+For help with developing plugins, see the [plugin development resources](#more-information).
+  
+    If you have a custom plugin written in a language other than Lua, convert the schema 
+into a `schema.lua` file before uploading it to {{site.konnect_short_name}}.
+
+* If using the `/plugin-schemas` API, you have a personal access token or a system account
+token to authenticate with the API. You can pass your token with any API request using an 
+authentication header:
+
+```
+--header 'Authorization: Bearer kpat_xgfT'
+```
+
+## Add a custom plugin to a {{site.konnect_short_name}} org
+
+{{site.konnect_short_name}} only requires the custom plugin's `schema.lua` file. 
+Using that file, it creates a plugin entry in the plugin catalog for your control plane.
+
+Upload a custom plugin schema to create a configurable entity in {{site.konnect_short_name}}:
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. From the **Gateway Manager**, open a control plane.
+1. Open **Plugins** from the side navigation, then click **Add Plugin**.
+1. Open the **Custom Plugins** tab, then click **Create** on the Custom Plugin tile.
+1. Upload the `schema.lua` file for your plugin.
+1. Check that your file displays correctly in the preview, then click **Save**.
+
+{% endnavtab %}
+{% navtab Konnect API %}
+
+Upload the `schema.lua` file for your plugin using the `/plugin-schemas` endpoint:
+
+```sh
+curl -i -X POST \
+  https://{region}.api.konghq.com/v2/{controlPlaneId}/core-entities/plugin-schemas \
+  --data lua_schema=@example-schema.lua
+```
+
+This example specifies a file, but you can also include the entire schema in the request as JSON.
+
+You should get an HTTP 201 response. You can check that your schema was uploaded using the following request:
+
+```sh
+curl -i -X GET \
+  https://{region}.api.konghq.com/v2/{controlPlaneId}/core-entities/plugin-schemas
+```
+
```

Comment:
What will I receive in that request that is different from the 201 I got in the request before? Maybe add:

"The request will return a JSON object representation of your plugin if it was updated correctly"